tests/RouteTest.php

```diff
@@ -331,6 +331,22 @@ public function a_route_prefix_can_be_registered_before_the_locale(): void
         $this->assertNotNull(route('prefix.fr.test'));
     }
 
+    /** @test **/
+    public function a_route_with_defaults_parameters_can_be_registered(): void
+    {
+        $params = ['param_1'=>'value_1', 'param_2'=>'value_2'];
+        Route::multilingual('test')->defaults($params)->name('test');
+
+        foreach (config('locales.supported') as $locale) {
+            $route = \Route::getRoutes()->getByName($locale.'.test');
```

Comment:
No need the `\` since we are already importing the Route facade.

Reply:
Done !
Just waiting the merge to send the second pull request for the second feature :)